src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/NcksToCsv.java

```diff
@@ -0,0 +1,176 @@
+package au.org.emii.gogoduck.worker;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class NcksToCsv {
```

Comment:
I don't understand why there is another class. This implementation belongs to `CSVConverter`. Unless you *really* want it in this class.